pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -318,7 +321,9 @@ func (gb *GraphBuilder) Run(stopCh <-chan struct{}) {
 }
 
 var ignoredResources = map[schema.GroupResource]struct{}{
-	{Group: "", Resource: "events"}: {},
+	{Group: "", Resource: "events"}:                     {},
+	{Group: eventv1.GroupName, Resource: "events"}:      {},
+	{Group: eventv1beta1.GroupName, Resource: "events"}: {},
```

Comment:
According to the statement here eventv1beta1 is deprecated in 1.22.So I guess v1 is enough.
https://github.com/kubernetes/kubernetes/blob/89aaf7c02dd33415ac994cecfddca7c02ed75834/staging/src/k8s.io/api/events/v1beta1/types.go#L27

Reply:
ok, i have removed the v1beta1 version, thx